client/src/components/NeighborhoodMap.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Details from './Details';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+const googleMapApi = require('../../../googleMapConfig');
+
+class NeighborHoodMap extends React.Component {
+  constructor(props) {
+    super(props);
+
+  }
+
+
+  render() {
+    const mapStyles = {
+      width: '50%',
+      height: '50%',
+    };
+
+    const { lat, lng } = this.props;
+
+    return (
+        <Map
+          google={this.props.google}
```

Comment:
you're using this.props here (line 23) when you're restructuring this.props.lat and lang on line 19. Just throw this.props.google in to that same destructured line (19) and just write 'google' on line 23 - remove this.props to stay consistent.

Reply:
I'm afraid this might interfere with how the google map component functions.